packages/svelte/src/compiler/phases/2-analyze/visitors/ConstTag.js

```diff
@@ -12,6 +13,13 @@ export function ConstTag(node, context) {
 		validate_opening_tag(node, context.state, '@');
 	}
 
+	for (let declaration of node.declaration.declarations) {
+		const rune = get_rune(declaration.init, context.state.scope);
+		if (rune) {
+			e.const_tag_invalid_rune_usage(declaration.init, rune);
```

Comment:
should we start with the more generic "rune X cannot be used here" errors that already exist? that way we don't need to invent another error code

Reply:
Oh i knew we had an error like this...i tried to search for it and miserably failed 😄

Reply:
Oh actually we don't have a generic one...we only have specific ones (i think)...i'll check better after TWIS!

Reply:
Yeah confirmed there's no generic `rune X cannot be used here`...should we create a generic error like this instead of the specific one?

Reply:
My thinking was that you check which rune it is and then throw one of the more specific "X can only be used at ..." errors - I haven't checked the error message itself though, so if you find that's not possible then yes maybe a more generic error message.

Reply:
Yeah the other error messages are very specific... I'll create a rune_cannot_be_used error